app/models/manageiq/providers/ansible_tower/embedded_ansible_worker.rb

```diff
@@ -0,0 +1,20 @@
+class ManageIQ::Providers::AnsibleTower::EmbeddedAnsibleWorker < MiqWorker
```

Comment:
@durandom Running ansible locally on the server (embedded) and talking to it through an endpoint are fundamentally different and, in my opinion, not actually related at all. They will share no common functionality.

I think what we need to do is come up with a new template for what we call a thing that actually is responsible for running a service locally (because to me, that's not a provider at all).